endogenous/Snakefile

```diff
@@ -0,0 +1,11 @@
+configfile: "config.yaml"
+
+wildcard_constraints:
+    simpl="[a-zA-Z0-9]*",
+    clusters="[0-9]+m?"
+    
+rule transport:
+    input:
+        transport_demand=expand("resources/transport_demand_s{simpl}_{clusters}.csv",**config['scenario'])
```

Comment:
If wildcard/regular expression _{simpl}_ is removed (see above comment), then its invocation has to be removed from this line too.